torchci/rockset/commons/__sql/recent_pr_workflows_query.sql

```diff
@@ -1,20 +1,23 @@
-WITH recent_shas as (
-    SELECT
-        p.head.sha as sha,
-        p.number as number
-    FROM
-        workflow_job j
-        JOIN commons.pull_request p ON j.head_sha = p.head.sha
-    where
-        (
+WITH
+    recent_shas as (
+        SELECT
+            p.head.sha as sha,
+            p.number as number,
+            PARSE_TIMESTAMP_ISO8601(push.head_commit.timestamp) as sha_time,
+        FROM
+            workflow_job j
+            JOIN commons.pull_request p ON j.head_sha = p.head.sha
+            JOIN commons.push push on push.head_commit.id = j.head_sha
+        where
```

Comment:
Why this is needed?

Reply:
Same thing - I just pulled the changes from Rockset and committed them.

Reply:
Figured it out offline: somebody made changes to the query, which were not deployed to production and as result got pulled into this PR. Updated fixed it.